libs/src/api/serviceProvider.js

```diff
@@ -75,17 +73,13 @@ class ServiceProvider extends Base {
    */
   async autoSelectCreatorNodes ({
     numberOfNodes = 3,
-    whitelist = null,
```

Comment:
would we need to change this? https://github.com/AudiusProject/audius-protocol/blob/master/libs/src/sanityChecks/rolloverNodes.js#L52-L54

Reply:
good spot. fixing